vllm/v1/sample/metadata.py

```diff
@@ -41,3 +41,8 @@ class SamplingMetadata:
 
     # req_index -> bad_words_token_ids
     bad_words_token_ids: dict[int, list[list[int]]]
+
+
+@dataclass
+class PoolingMetadata:
```

Comment:
where is this used?

Reply:
It's imported as `V1PoolingMetadata` in `pooler.py`